README.md

```diff
@@ -138,6 +138,10 @@ python3 setup.py develop
 
 Please see [setup.py](setup.py).
 
+### Image build
+
+In order to speed up frequent builds and avoid issues with dependencies, docker image makes use [`qontract-reconcile-build`](https://quay.io/repository/app-sre/qontract-reconcile-base?tag=latest&tab=tags) image. See [`app-sre/coontainer-images`](https://github.com/app-sre/container-images) repository if you want to make changes to the base image. This repo [`Dockerfile`](dockerfiles/Dockerfile) must only contain only instructions related to the python code build.
```

Comment:
Btw, let's please break the line. A single line break isn't rendered as such and makes diffs nicer.

Reply:
fixed in 6252117